### PR TITLE
Misc stuff

### PR DESCRIPTION
**Pull request by vespian**

### CODE REVIEW

**Comment by vespian** [comment]
@alberts @jfrazelle ready for review!

File: Makefile [hunk 1]
```diff
@@ -205,7 +205,7 @@ clean-containers: ## Removes and cleans up the master, agent, and installer cont
 	@$(foreach NUM,$(shell seq 1 $(AGENTS)),$(call remove_container,$(AGENT_CTR),$(NUM)))
 
 clean-slice: ## Removes and cleanups up the systemd slice for the mesos executor.
-	@sudo systemctl start mesos_executors.slice
+	@sudo systemctl stop mesos_executors.slice
```
Comment (jfrazelle) [inline Makefile]: nice :)

**Comment by jfrazelle** [comment]
LGTM

**Comment by vespian** [comment]
thanks @jfrazelle :) merging!

**Pull request merged**